pkg/apiserver/vshn/redis/vshnredis.go

```diff
@@ -24,5 +24,16 @@ func (c *concreteRedisProvider) ListVSHNRedis(ctx context.Context, namespace str
 		return nil, err
 	}
 
+	cleanedList := make([]vshnv1.VSHNRedis, 0)
+	for _, p := range instances.Items {
```

Comment:
Shouldn't this be symetric to the `ListXVSHNPostgreSQL()` function? Nothing in the method name or docs indicates that this list is filtered by a random field. I would have put it here naively: https://github.com/vshn/appcat-apiserver/blob/3549ec84e7e3c19c1f28763ae2e1257e41391151/pkg/apiserver/vshn/redis/list.go#L34

Also it is never regression tested.